.vscode/launch.json

```diff
@@ -5,19 +5,18 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "Launch test/e2e/app-dir/app development",
+      "name": "Launch app development",
       "type": "node",
       "request": "launch",
       "cwd": "${workspaceFolder}",
+      "webRoot": "examples/hello-world",
```

Comment:
VS Code does warn `Property webRoot is not allowed.` here because the property is only meant for chrome launch configs, but it does work nicely for referencing it further down with `${webRoot}`. This gives us a single place to adjust when choosing a different app.

Reply:
I'd opt for copy&past if it already warns. VSCode may start ignoring it. There's no support for arbitrary variables in these configs?

Reply:
Only `workspaceFolder` and `webRoot` are [resolved](https://github.com/microsoft/vscode-js-debug/blob/7349abcd0aaf72375645a4a876afab6479e0ed7e/src/configuration.ts#L1121-L1126). It feels a bit error prone if one needs to adjust all three occurrences (in `runtimeArgs`, and twice in `sourceMapPathOverrides`) when switching to a different app. We would also need to explain that in [contributing/core/vscode-debugger.md](https://github.com/vercel/next.js/pull/66254/files#diff-6b75128a13efad78be9d1b1a2423a1118d4f1859c9ec2d0114094037bc012a00R19).

Reply:
How about we use [input variables](https://code.visualstudio.com/docs/editor/variables-reference#_input-variables) instead? see [cb2810b](https://github.com/vercel/next.js/pull/66254/commits/cb2810bf38b773d531ff9900766a790a958dcaa2)